packages/apply/src/index.ts

```diff
@@ -245,7 +245,7 @@ const parentBump = (
           };
           if (changes[pkg].changes) {
             changes[pkg].changes!.forEach((parentChange) => {
-              parentChange.meta.dependencies = `Bumped due to a bump in ${main}.`;
+              parentChange.meta.dependencies = main;
```

Comment:
Here is where I am having issues, I noticed that the `meta.dependencies` will be set only if the package itself doesn't have changes on its own. And when it doesn't have changes, This is seems to be called once only even though the package has two dependencies which are bumped at the same time.

Reply:
This affects the logic here https://github.com/jbolda/covector/pull/243/files#diff-685473c979a3aff3d3a5a5e1323dd7f4c5bb073c8462ce72b30a6c03494e05a2R210-R239 which is responsible for nesting the dependencies changes in the parent under groups as described in the PR body

Reply:
I am not familiar with the codebase so I need help understanding the flow of adding dependencies changes to the parent.

Reply:
I think at a high level, we want that bump in a child to bump the children (which I think we presume is a patch). I think the idea was to add a bump if it didn't have one, but I don't know how keen I am with the current logic if I am being honest.

I think maybe it still makes sense to "trickle" that change text up to the parent, but..... maybe not?

Reply:
No, I think it is fine to bump the parent when it doesn't have a bump already but I have to questions:
1. Should the children bump be rendered in the parent at all time or only if parent doesn't have one? I think it should be always rendered but minified like this:
	
	![image](https://user-images.githubusercontent.com/48618675/197529602-b87e05ad-8567-4b8b-9158-48c9bf42dabf.png)
	
	also with #244 it will be under its own section "Dependencies"

2. I noticed that if the parent doesn't have a change file, it will only use one of the children as the bump and not all children? am I correct or did I miss something? shouldn't all children bumps be rendered?

<sub>The parent here is a package that has a dependency on child1 and child2 </sub>

Reply:
Alright, I ended up always including all children bumps if any.

Reply:
Yea, I think it only uses one child, but it really should have used all of them.

I like the thought of the minified reference to the child bump, but I believe that is very specific to Github rendering and wouldn't look great in the `CHANGELOG.md`. I would prefer to stick to default markdown rendering specs I think. Really the idea here is that we are defaulting to a patch bump because we have a dependency on that package and want to include the new version.

The logic currently only applies to pinned deps, but I want to extend it into considering ranges as well which I think makes this more valuable. Considering all of this, the value here is giving the consumer a way to tie back to the package that caused this bump and see what changed. In reality, maybe the better option is a markdown link to the changelog for the version that this was bumped to? Something like this perhaps?

> Bumped due to an upgrade of [`package-utils` to v2.2.1](https://github.com/jbolda/covector/CHANGELOG.md#2.2.1).

Reply:
Would it make sense to fix this to consider all children, but deal with the rendering in a separate PR perhaps?

Reply:
I do like the idea linking to its release but that needs to get the git release link which is a bit more involved. Also we can just do:
> Bumped due to an upgrade of `package-utils` 

Without linking or `<details>`

Reply:
Yea that works for me. Yea, the flow of some of it could definitely use some work. Been slowly handling it, but... ya know 😅 .

Ready for me to review the code?

Reply:
yeah go ahead, thanks